modules/rank-eval/src/main/java/org/elasticsearch/index/rankeval/ReciprocalRank.java

```diff
@@ -45,7 +45,7 @@
 public class ReciprocalRank implements RankedListQualityMetric {
 
     public static final String NAME = "reciprocal_rank";
-    public static final int DEFAULT_MAX_ACCEPTABLE_RANK = 10;
```

Comment:
Yes, thats the other option. I'm going to remove this so the maximum acceptable rank is controlled by the underlying search request size.